examples/p4est_2d_dgsem/elixir_euler_forward_step.jl

```diff
@@ -0,0 +1,155 @@
+basis = LobattoLegendreBasis(polydeg)
+shock_indicator = IndicatorHennemannGassner(equations, basis,
+                                            alpha_max=0.5,
+                                            alpha_min=0.001,
+                                            alpha_smooth=true,
+                                            variable=density_pressure)
+volume_integral = VolumeIntegralShockCapturingHG(shock_indicator;
+                                                 volume_flux_dg=volume_flux,
+                                                 volume_flux_fv=surface_flux)
+solver = DGSEM(polydeg=polydeg, surface_flux=surface_flux, volume_integral=volume_integral)
+
+# Get the unstructured quad mesh from a file (downloads the file if not available locally)
+default_mesh_file = joinpath(@__DIR__, "abaqus_forward_step.inp")
+isfile(default_mesh_file) || download("https://gist.githubusercontent.com/andrewwinters5000/b346ee6aa5446687f128eab8b37d52a7/raw/511917dfffcc2347d844003ec53142c5823bcbe7/abaqus_forward_step.inp",
+                                      default_mesh_file)
+mesh_file = default_mesh_file
+
+mesh = P4estMesh{2}(mesh_file)
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver,
+                                    boundary_conditions=boundary_conditions)
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+tspan = (0.0, 4.0)
+ode = semidiscretize(semi, tspan)
+
+summary_callback = SummaryCallback()
+
+analysis_interval = 1000
+analysis_callback = AnalysisCallback(semi, interval=analysis_interval,
+                                     extra_analysis_integrals=(entropy,))
+
+alive_callback = AliveCallback(analysis_interval=analysis_interval)
+
+save_solution = SaveSolutionCallback(interval=500,
+                                     save_initial_solution=true,
+                                     save_final_solution=true,
+                                     solution_variables=cons2prim)
+
+amr_indicator = IndicatorLöhner(semi, variable=density)
+
+amr_controller = ControllerThreeLevel(semi, amr_indicator,
+                                      base_level=0,
+                                      med_level=2, med_threshold=0.05,
+                                      max_level=5, max_threshold=0.1)
+
+amr_callback = AMRCallback(semi, amr_controller,
+                           interval=5,
+                           adapt_initial_condition=true,
+                           adapt_initial_condition_only_refine=true)
+
+callbacks = CallbackSet(summary_callback,
+                        analysis_callback, alive_callback,
+                        save_solution,
+                        amr_callback)
+
+# positivity limiter necessary for this example with strong shocks
+stage_limiter! = PositivityPreservingLimiterZhangShu(thresholds=(5.0e-6, 5.0e-6),
+                                                     variables=(Trixi.density, pressure))
```

Comment:
```suggestion
                                                     variables=(density, pressure))
```
We also export `density`. Or is there another reason you need the fully qualified name here?

Reply:
I borrowed this call from the astro jet elixir. I can change this

Reply:
But you should try to be consistent and use the prefix everywhere (there is another `density` above) or not at all to avoid confusing people reading this nice elixir

Reply:
Done. I now consistently use `Trixi.density`